alws/routers/build_node.py

```diff
@@ -94,12 +94,18 @@ async def get_task(
             "email": task.build.owner.email,
         },
     }
+    supported_arches = request.supported_arches
+    if 'src' in request.supported_arches:
+        supported_arches.remove('src')
+    task_arch = task.arch
+    if task_arch == 'src':
+        task_arch = supported_arches[0]
```

Comment:
Okay, but what will happen when you finishing src task? It looks like this way you'll be treating src task like e.g. aarch64 task which is not fully correct

Reply:
No that is exactly what I am trying to do here. This part of the code adds repositories for a builder to build RPM. So because we can build `src`  on any builders we need to add here only repositories for builder supported arch.